lib/rewards.js

```diff
@@ -11,10 +11,13 @@ export const runUpdateRewardsLoop = async ({ contracts, ethAddress, onMetrics })
     while (!contracts.length) {
       await timers.setTimeout(1000)
     }
-    const contractRewards = await Promise.all(contracts.map(async contract => {
-      return getScheduledRewardsWithFallback(contract, ethAddress)
-    }))
-    const totalRewards = contractRewards.reduce((a, b) => a + b, 0n)
+    const rewards = await Promise.all([
+      ...contracts.map(async contract => {
+        return getContractScheduledRewardsWithFallback(contract, ethAddress)
+      }),
+      getSparkRewardsScheduledRewardsWithFallback(ethAddress)
```

Comment:
This name is awkward but correct. @bajtos do you have a better suggestion?

Reply:
How about `getOffchainScheduledRewardsWithFallback`?